app/schemas/user.js

```diff
@@ -0,0 +1,25 @@
+exports.signup = {
+  type: 'object',
+  properties: {
+    name: {
+      type: 'string',
+      minLength: 1
+    },
+    last_name: {
+      type: 'string',
+      minLength: 1
+    },
+    email: {
+      type: 'string',
+      format: 'email',
+      pattern: '@wolox.(com(.ar|.co|.cl|)|ar|co|cl)$'
```

Comment:
This could be a constant, please create a constants file a put this type of constants there